less/transferwise.less

```diff
@@ -36,6 +36,27 @@ address {
 }
 .footer {
     background-color: @gray-base;
+
+	.icon {
+		font-size: 50px !important;
+		margin: 0 5px 0 5px;
```

Comment:
Not sure we need this.  We have some `.icon-lg .icon-xl .icon-xxl` utility classes.  If we do need it, I would expect it to have a total height that's divisible by 8 :-)